### DISCUSSION TITLE
Suggestions for tweaks/improvements

### DISCUSSION THREAD

**Discussion by ColonelGerdauf** [PrismLauncher/PrismLauncher#957] [Ideas]
The (admittedly big) list is as follows:

- move shader packs to a singular directory that each instance can use, and have the minecraft instance runtime find the shaders in that directory
- update function for shader, texture, and resource packs
- if there are mods that do not match the correct version, give the listing a colour for it (and maybe distinguish a loose below match and no match)
- provide a way to have a simple "latest stable" and "latest devbuild" that auto-upgrades the minecraft versions, as well as auto-update any mods and packs, and disables kits that have yet to be updated (though update check remains valid)
- in the case of latest devkit, the save-data does not need to be saved to disk. 
- have a notification if there is any incompatible mods or packs loaded in an instance (for example: a small exclamation mark at the top left corner of the image)
- a mechanism to do a simple "update everything in instance", including a separate mechanism to mass-update mods in any specific instance
- add a column in the mods table to show what platform each mod is linked to (including manual, for which update has to be manual and specific)
- have the Prism packages come packaged with the needed Java runtimes as sandboxes
- allow parameters to be set as the default config for game launches (including base game, Optifine, Sodium, etc)
- have Prism handle Modrinth and CurseForge links internally when looking at mod entries
- remove the mechanism where mods do not update when unchecked (disabled)
- allow playback of GIF's in the mod details
- improve the speed of mass updates from the CurseForge end where possible
- provide a section for donation linking in the case of Modrinth, where applicable. This can be enabled/disabled in Prism settings.
- update mods by changing the file data instead of file swapping
- update the launcher to make use of the released LWJGL 3.3.2 and later
- in the lists, provide a counter somewhere that counts the number of mods, core files, and packs

**Comment by TheKodeToad** [comment]
hmm, why not an issue?

**Reply by ColonelGerdauf** [reply]
[repost because I was stupid] Because it is a long list that can get lost in the details, and I was more so looking for feedback about if any in particular is unfeasible.

**Comment by Poopooracoocoo** [comment]
Just went through and searched for any issues for your suggestions if there were any. You should create issues for the ones listed as "n/a"

1. #38
2. #153
3. #840 
4. n/a
5. n/a
6. n/a
7. n/a
8. n/a
9. sounds like you're either suggesting MSIX or automatic java downloading. #995 #734 
10. n/a
11. #1019
12. n/a
13. n/a
14. n/a
15. n/a
16. n/a
17. n/a
18. #451

**Comment by Universal-Flare** [comment]
Could you add My Suggested Improvements for Mod Installation -

- Add more Filters for the Mod Selection Menu like a Filter for Non-Installed Mods , and other filters from Modrinth or other platforms to make it easier for installation .
- Add Version sorting for the Instance Menus .

**Comment by RyanLatour** [comment]
Can favorite mods be added to the Prism Launcher?
so you don't need to have a list elsewhere and search for them ever time you make a new instance?

**Reply by Universal-Flare** [reply]
well you could just copy and paste the mods form another instance right ?
